src/Literate.jl

```diff
@@ -75,8 +76,9 @@ function parse(content; allow_continued = true)
             if !(chunks[end] isa CodeChunk)
                 push!(chunks, CodeChunk())
             end
-            # remove "## " and "##\n"
-            line = replace(replace(line, r"^(\h*)#(# .*)$" => s"\1\2"), r"^(\h*#)#$" => s"\1")
+            # remove "## " and "##\n", strip the leading "#" from "## xyz" and "##| xyz"
```

Comment:
If you expand a bit on these new tokens in the documentation (probably under a new Quarto section under Markdown output) I can merge this.

(Sorry it took some time, just got back from vacation (https://github.com/fredrikekre/Literate.jl/pull/200#issuecomment-1223985373) :p )